<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Background</class>
 <widget class="QWidget" name="Background">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>458</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>275</width>
    <height>458</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>275</width>
    <height>458</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Rigeru</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>assets/icon.jpg</normaloff>assets/icon.jpg</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#151b1f;</string>
  </property>
  <widget class="QCheckBox" name="checkboxMale">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>78</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>RomanC</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ClosedHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#1e2329;
color:#797979;</string>
   </property>
   <property name="text">
    <string>Male</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkboxFemale">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>RomanC</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ClosedHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#1e2329;
color:#797979;</string>
   </property>
   <property name="text">
    <string>Female</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkboxMuslim">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>78</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>RomanC</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ClosedHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#1e2329;
color:#797979;</string>
   </property>
   <property name="text">
    <string>Muslim</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkboxProtestant">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>RomanC</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ClosedHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#1e2329;
color:#797979;</string>
   </property>
   <property name="text">
    <string>Protestant</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkboxCatholic">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>78</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>RomanC</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ClosedHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#1e2329;
color:#797979;</string>
   </property>
   <property name="text">
    <string>Catholic</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkboxHindu">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>RomanC</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ClosedHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#1e2329;
color:#797979;</string>
   </property>
   <property name="text">
    <string>Hindu</string>
   </property>
  </widget>
  <widget class="QLabel" name="background2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>231</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#1e2329;
border-radius: 8px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="background1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#1e2329;
border-radius: 8px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelGender">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>65</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>RomanC</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#1e2329;
color:#797979;</string>
   </property>
   <property name="text">
    <string>Gender</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelReligion">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>145</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>RomanC</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#1e2329;
color:#797979;</string>
   </property>
   <property name="text">
    <string>Religion</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>15</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>RomanC</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#797979;</string>
   </property>
   <property name="text">
    <string>DTMM Automated Line </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="seperator1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>275</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#797979;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="seperator2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>275</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#797979;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="background3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>231</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#1e2329;
border-radius: 8px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="textInput">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>RomanC</family>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#2f3237;
color:#797979;
border-radius: 5px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'RomanC'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="placeholderText">
    <string>Write message here ...  </string>
   </property>
  </widget>
  <widget class="QPushButton" name="startProgram">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>231</width>
     <height>24</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#1e2329;
color:#797979;
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>Start Program</string>
   </property>
  </widget>
  <widget class="QLabel" name="seperator3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>400</y>
     <width>275</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#797979;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="selectFile">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ClosedHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#2f3237;
color:#797979;
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>Select File</string>
   </property>
  </widget>
  <zorder>background1</zorder>
  <zorder>background2</zorder>
  <zorder>checkboxMale</zorder>
  <zorder>checkboxFemale</zorder>
  <zorder>checkboxMuslim</zorder>
  <zorder>checkboxProtestant</zorder>
  <zorder>checkboxCatholic</zorder>
  <zorder>checkboxHindu</zorder>
  <zorder>labelGender</zorder>
  <zorder>labelReligion</zorder>
  <zorder>labelTitle</zorder>
  <zorder>seperator1</zorder>
  <zorder>seperator2</zorder>
  <zorder>background3</zorder>
  <zorder>textInput</zorder>
  <zorder>startProgram</zorder>
  <zorder>seperator3</zorder>
  <zorder>selectFile</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
